src/ui/popup/settings-page/mange-settings-button.tsx

```diff
@@ -0,0 +1,18 @@
+import {m} from 'malevic';
+import {NavButton} from '../../controls';
+import ControlGroup from '../control-group';
+
+export default function ManageSettingsButton(props: {onClick: () => void}) {
+    return (
+        <ControlGroup>
+            <ControlGroup.Control>
+                <NavButton onClick={props.onClick}>
+                    Manage Settings
+                </NavButton>
+            </ControlGroup.Control>
+            <ControlGroup.Description>
+                Manage your current settings.
```

Comment:
The current description is too similar to button text, use `Reset, export or import settings`, dot is unnecessary